### PR TITLE
Publish news and comms finder email signup

### PR DESCRIPTION
**Pull request by bilbof**
https://trello.com/c/HZHHKXc8/311-make-it-possible-to-publish-news-and-communications-email-alert-content-item-from-rummager

**Note:** this will need to be merged after https://github.com/alphagov/govuk-content-schemas/pull/860 is merged and the latest version of the gem is used by rummager. The tests wont pass until this is done.

This adds `ContentItemPublisher` to handle the publication of finders and their email signup pages from rummager. Additionally it adds a rake task to publish the content items.

### CODE REVIEW

File: lib/content_item_publisher/finder_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module ContentItemPublisher
+  class FinderPresenter < ContentItemPresenter
+    def present_links
+      links = {}
+      links["email_alert_signup"] = email_alert_signup if email_alert_signup?
```
Comment (koetsier) [inline lib/content_item_publisher/finder_presenter.rb:5]: could make line 5 and 6 more consistent. - they do a similar thing but are implemented differently

**Review by sophiebrown01 — APPROVED** [review]
Discussed with @bilbof

File: lib/content_item_publisher/publisher.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    def call
+      publish(content_item)
+    end
+
+  private
+
+    attr_reader :content_item, :logger, :timestamp
+
+    def publishing_api
```
Comment (koetsier) [inline lib/content_item_publisher/publisher.rb]: Could use Services.publishing_api here

File: spec/unit/content_item_publisher/finder_email_signup_publisher_spec.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  describe "#call" do
+    let(:publishing_api) { instance_double("GdsApi::PublishingApiV2") }
+    let(:payload) {
+      ContentItemPublisher::FinderEmailSignupPresenter.new(finder, timestamp).present
+    }
+
+    before do
+      allow(logger).to receive(:info)
+      allow(GdsApi::PublishingApiV2).to receive(:new).and_return(publishing_api)
```
Comment (koetsier) [inline spec/unit/content_item_publisher/finder_email_signup_publisher_spec.rb]: - Line 24 could be a bit different when using Services.publishing_api
- Could think about using test helpers defined in: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/test_helpers/publishing_api_v2.rb

**Pull request merged**